### DISCUSSION TITLE
Frohe Weihnachten

### DISCUSSION THREAD

**Discussion by sidey79** [RFD-FHEM/RFFHEM#922] [General]
@HomeAutoUser 
@elektron-bbs 

Wir waren nicht immer der gleichen Meinung im letzten Jahr, aber ich denke wir haben es doch immer hinbekommen uns zu verständigen.

Frohe Weihnachten und bleibt gesund.

**Comment by elektron-bbs** [comment]
Das wäre auch langweilig, wenn immer alle der gleichen Meinung wären :-)

Auch von mir: Schöne Feiertage

**Comment by HomeAutoUser** [comment]
Ich wünsche Euch und euren Familien ebenso ein schönes Fest noch ;)

Das wir weiterhin auch immer abwechslungsreiche Eingebungen und geistige Ergüsse haben :D

**Reply by sidey79** [reply]
@HomeAutoUser 
@elektron-bbs 

Frohe Weihnachten

**Reply by HomeAutoUser** [reply]
Das selbige wünsche ich auch Dir @sidey79 und allen anderen. Auf neue kreative Ideen ;)

**Reply by elektron-bbs** [reply]
Von mir natürlich auch an alle: Schöne Weihnachten!

**Comment by sidey79** [comment]
@HomeAutoUser 
@elektron-bbs 

Der Post ist wieder aktuell :)

Frohe Weihnachten.

**Reply by elektron-bbs** [reply]
Schon wieder ein Jahr vorüber :-)

Schöne Weihnachtstage euch allen!

**Comment by HomeAutoUser** [comment]
Auch von mir Frohe Weihnachten :)
